test/regressions/vite.config.js

```diff
@@ -15,6 +19,39 @@ export default defineConfig({
         import.meta.dirname,
         '../../node_modules/@mui/monorepo/packages/mui-docs/src',
       ),
+      '@mui/x-data-grid': path.resolve(import.meta.dirname, '../../packages/x-data-grid/src'),
+      '@mui/x-data-grid-generator': path.resolve(
+        import.meta.dirname,
+        '../../packages/x-data-grid-generator/src',
+      ),
+      '@mui/x-data-grid-pro': path.resolve(
+        import.meta.dirname,
+        '../../packages/x-data-grid-pro/src',
+      ),
+      '@mui/x-data-grid-premium': path.resolve(
+        import.meta.dirname,
+        '../../packages/x-data-grid-premium/src',
+      ),
+      '@mui/x-date-pickers': path.resolve(import.meta.dirname, '../../packages/x-date-pickers/src'),
+      '@mui/x-date-pickers-pro': path.resolve(
+        import.meta.dirname,
+        '../../packages/x-date-pickers-pro/src',
+      ),
+      '@mui/x-charts': path.resolve(import.meta.dirname, '../../packages/x-charts/src'),
+      '@mui/x-charts-pro': path.resolve(import.meta.dirname, '../../packages/x-charts-pro/src'),
+      '@mui/x-tree-view': path.resolve(import.meta.dirname, '../../packages/x-tree-view/src'),
+      '@mui/x-tree-view-pro': path.resolve(
+        import.meta.dirname,
+        '../../packages/x-tree-view-pro/src',
+      ),
+      '@mui/x-license': path.resolve(import.meta.dirname, '../../packages/x-license/src'),
+      '@mui/x-telemetry': path.resolve(import.meta.dirname, '../../packages/x-telemetry/src'),
+      '@mui/x-internals': path.resolve(import.meta.dirname, '../../packages/x-internals/src'),
+      '@mui/material-nextjs': path.resolve(
+        import.meta.dirname,
+        '../../node_modules/@mui/monorepo/packages/mui-material-nextjs/src',
+      ),
+      docs: path.resolve(import.meta.dirname, '../../node_modules/@mui/monorepo/docs'),
```

Comment:
I have tried to move away from alias lists. If we do want them back, we should probably find a way to avoid having multiple copies of it.

Reply:
Without this, we have to build the whole project and have it in watch mode to debug/run the visual regression tests, which feels super slow (+200s on my end). #17584

https://github.com/mui/base-ui/blob/a7451a08424f12d913c8355cccf3274ef5f3e055/package.json#L38 or https://github.com/mui/material-ui/blob/a705e1f15075b2deb59263868bfa7b1d9f84cdd4/package.json#L75 seems better.